src/polymer/content-to-slot-declarations.ts

```diff
@@ -75,6 +75,23 @@ class ContentToSlotDeclarations extends HtmlRule {
               range: parsedDocument.sourceRangeForEndTag(contentElement)!
             }
           ];
+          if (isSafe) {
+            warning.fix = edit;
+          } else {
+            warning.actions = [{
+              kind: 'edit',
+              code: 'content-with-select',
+              description: stripIndent(`
+                Convert this <content> to a <slot>.
+
+                This changes the API of this element because the \`select\`
+                will become a slot name. Use the content-to-slot-usages lint
+                pass to convert usages of the element to conform to the new
+                API.
+            `).trim(),
```

Comment:
it'd be nice if this was built into stripIndent, and as a template tag.

Reply:
Added a new util method `stripIndentation` to do both. I didn't make it a template tag as I didn't want to deal with the hassle of interpolation. Should we prefer functions on a single string to be template tags?

Reply:
I think it's just a bit nicer ergonomically, but then again it doesn't compose with other tags very easily, so 🤷‍♂️